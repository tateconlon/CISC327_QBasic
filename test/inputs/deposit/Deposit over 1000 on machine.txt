login
machine
deposit
1234567
5000000
logout