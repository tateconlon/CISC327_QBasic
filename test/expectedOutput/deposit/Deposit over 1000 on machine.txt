login as machine successful
Cannot deposit more than $1000.00 in a single transaction in machine mode
Transaction summary written and system logged out successfully